CMakeLists.txt

```diff
@@ -251,10 +251,12 @@ if (WIN32)
   endif()
 endif()
 
```

Comment:
Can you make it

if(APPLE)
... your new line / not even sure it's needed.
elsif()
... the old stuff

Reply:
Sorry, you're right - it wasn't needed. I've updated it now.